frontend/src/components/common/button/Button.tsx

```diff
@@ -0,0 +1,104 @@
+import styled from '@emotion/styled';
+import React from 'react';
```

Comment:
You do not need to import `React`. It is only being used for types, but typescript can find it without importing